public-interface/iot-entities/postgresql/models/devices.js

```diff
@@ -49,7 +49,6 @@ module.exports = function (sequelize, DataTypes) {
                 allowNull: false,
                 defaultValue: 'created'
             },
```

Comment:
I think the comma leaves invalid JSON if you remove the lastVisit element. If you agree, can you please remove it?